sphinx_book_theme/static/images/logo_deepnote.svg

```diff
@@ -0,0 +1 @@
 1.047h-.091V5.868h-1.144V14z" fill="#fff"/><path d="M63.043 14h1.15V8.15h-1.15V14zm.575-6.864c.403 0 .73-.339.73-.752a.744.744 0 00-.73-.758.744.744 0 00-.73.758c0 .413.327.752.73.752zM65.69 14h1.155v-3.421c0-.913.542-1.483 1.396-1.483.854 0 1.263.467 1.263 1.407V14h1.154v-3.787c0-1.39-.72-2.175-2.025-2.175-.88 0-1.46.392-1.745 1.042h-.086v-.93H65.69V14zm9.338-7.75V14h2.798c2.31 0 3.647-1.423 3.647-3.9 0-2.438-1.348-3.85-3.647-3.85h-2.798zm1.203 1.095h1.46c1.607 0 2.558 1.02 2.558 2.772 0 1.777-.935 2.787-2.557 2.787H76.23V7.345zm8.942 1.676c.854 0 1.412.596 1.439 1.515h-2.954c.064-.913.66-1.515 1.515-1.515zm1.434 3.4c-.215.456-.693.709-1.391.709-.924 0-1.52-.65-1.558-1.676v-.064h4.13v-.403c0-1.837-.983-2.949-2.61-2.949-1.649 0-2.696 1.187-2.696 3.056 0 1.87 1.025 3.019 2.701 3.019 1.338 0 2.283-.65 2.535-1.692h-1.111zm4.768-3.4c.854 0 1.412.596 1.44 1.515H89.86c.065-.913.66-1.515 1.515-1.515zm1.434 3.4c-.215.456-.693.709-1.391.709-.924 0-1.52-.65-1.558-1.676v-.064h4.13v-.403c0-1.837-.982-2.949-2.61-2.949-1.649 0-2.696 1.187-2.696 3.056 0 1.87 1.026 3.019 2.702 3.019 1.337 0 2.282-.65 2.535-1.692h-1.112zm5.402-4.367c-.795 0-1.482.398-1.837 1.064h-.086V8.15h-1.112v7.793h1.155v-2.83h.091c.307.617.967.983 1.8.983 1.477 0 2.417-1.166 2.417-3.024 0-1.859-.94-3.019-2.428-3.019zm-.328 5.006c-.966 0-1.573-.763-1.573-1.982 0-1.224.607-1.987 1.579-1.987.977 0 1.563.746 1.563 1.982 0 1.24-.586 1.987-1.569 1.987zm3.947.94h1.155v-3.421c0-.913.542-1.483 1.396-1.483.854 0 1.262.467 1.262 1.407V14h1.155v-3.787c0-1.39-.72-2.175-2.025-2.175-.881 0-1.461.392-1.745 1.042h-.086v-.93h-1.112V14zm8.866.113c1.708 0 2.756-1.144 2.756-3.04 0-1.886-1.053-3.035-2.756-3.035-1.708 0-2.76 1.15-2.76 3.035 0 1.896 1.047 3.04 2.76 3.04zm0-1.026c-1.004 0-1.568-.736-1.568-2.014 0-1.268.564-2.009 1.568-2.009.999 0 1.569.741 1.569 2.009 0 1.273-.57 2.014-1.569 2.014zm4.355-6.392v1.483h-.929v.972h.929v3.303c0 1.128.51 1.58 1.794 1.58.226 0 .44-.028.628-.06v-.961c-.161.016-.263.027-.44.027-.575 0-.827-.269-.827-.881V9.15h1.267v-.972h-1.267V6.695h-1.155zm5.918 2.326c.854 0 1.412.596 1.439 1.515h-2.954c.064-.913.661-1.515 1.515-1.515zm1.434 3.4c-.215.456-.693.709-1.391.709-.924 0-1.52-.65-1.558-1.676v-.064h4.13v-.403c0-1.837-.983-2.949-2.61-2.949-1.649 0-2.696 1.187-2.696 3.056 0 1.87 1.026 3.019 2.701 3.019 1.338 0 2.283-.65 2.536-1.692h-1.112z" fill="#B0B0B0"/></g><rect x=".5" y=".5" width="130.808" height="19" rx="2.5" stroke="#121416"/><defs><filter id="filter0_d" x="20.78" y="5.626" width="104.805" height="14.318" filterUnits="userSpaceOnUse" color-interpolation-filters="sRGB"><feFlood flood-opacity="0" result="BackgroundImageFix"/><feColorMatrix in="SourceAlpha" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"/><feOffset dy="2"/><feGaussianBlur stdDeviation="1"/><feColorMatrix values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.4 0"/><feBlend in2="BackgroundImageFix" result="effect1_dropShadow"/><feBlend in="SourceGraphic" in2="effect1_dropShadow" result="shape"/></filter></defs></svg>
```

Comment:
Could you change the SVG so that it's just the deepnote logo and not a button? Coding the whole SVG as a button makes it look out-of-place with the other launch buttons:

![image](https://user-images.githubusercontent.com/1839645/134221885-7e6edd68-24b7-41d3-b97d-1578b084fb13.png)

Reply:
Will do :)

Reply:
I pushed the logo fix now :)